lib/wallaby/partition_supervisor.ex

```diff
@@ -0,0 +1,565 @@
+#                                 Apache License
```

Comment:
@josevalim is it okay to vendor partition supervisor like this?

Reply:
Yes, I believe you need the copyright notice and a link to the repo.

Reply:
Here it is: https://github.com/elixir-lang/elixir/blob/main/NOTICE

the second bit.

Reply:
I'll add a link, thank you! I appreciate the clarification.